tests/tests_signin_user.py

```diff
@@ -0,0 +1,6 @@
+def test_with_authenticated_client(client, django_user_model):
+    username = "test"
+    password = "test"
+    django_user_model.objects.create_user(username=username, password=password)
+    respons = client.get('/')
+    assert respons.status_code == 200
```

Comment:
What are you trying to test here?
This code creates a new user and then checks if there is a main page.
This doesn't authenticate any user

Reply:
I thought it check the login procces

Reply:
It should be if you search for a hint in the rendered page, do you have some Title or header to check that User is really logged in?

Reply:
I don't believe that creating a new user will log in as that user.

Reply:
@simzacks You are correct, i think i missed the line in my talk
@BarBenNaim Please refer this
client.login(username=username, password=password)
https://pytest-django.readthedocs.io/en/latest/helpers.html#id2